src/Simulation/Native/CMakeLists.txt

```diff
@@ -36,10 +36,12 @@ option(USE_GATE_FUSION "Use gate fusion" ON)
 if (MSVC)
   # always create debug info
   add_definitions("/Zi")
+  add_definitions("/Z7")
 
   # build with no VC runtime depedencies:
-  set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /MT /Qspectre /guard:cf")
+  set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /MT /Qspectre /guard:cf /Zi /Z7")
   set(CMAKE_CXX_FLAGS_DEBUG   "${CMAKE_CXX_FLAGS_DEBUG}   /MTd /Qspectre /guard:cf")
+  set(CMAKE_SHARED_LINKER_FLAGS_RELEASE "${CMAKE_SHARED_LINKER_FLAGS_RELEASE} /DEBUG")
 else (MSVC)
```

Comment:
(Not a part of this PR changes. I don't know CMake and I can be mistaken)

It looks suspicious that `if` and `else` part both have `(MSVC)` in them:

if **(MSVC)**
...
else **(MSVC)**

Reply:
yeah, it's an odd convention to me too. Apparently it is optional, but quite common. It's similar to the way you often see comments on namespace close to clarify which scope is ending, as in
```c++
namespace Foo
{
namespace Bar
{
    ...
} // namespace Bar
} // namespace Foo
```